darwin/importer/formats/superannotate.py

```diff
@@ -0,0 +1,159 @@
+import json
+from functools import partial
+from os.path import join
```

Comment:
you probably don't need this import here (see later suggestions)

Reply:
fixed